src/multiselect_shortcut.js

```diff
@@ -184,15 +198,19 @@ const registerCut = function() {
     },
     callback: function(workspace) {
       copyData.clear();
+      const blockList = [];
       const apply = function(block) {
         if (cutShortcut.check(block)) {
           copyData.add(block.toCopyData());
-          block.workspace.hideChaff();
-          if (block.outputConnection) {
-            block.dispose(false, true);
-          } else {
-            block.dispose(true, true);
-          }
+          blockList.push(block.id);
+        }
+      };
+      const applyDelete = function(block) {
```

Comment:
For some reason I couldn't figure out, applyDelete does get called with block === null for me, resulting in an error. 

Adding `if(!block) return;` here solved that issue for me and made cutting work correctly.

Reply:
Thank you!